### DISCUSSION TITLE
How to change span color?

### DISCUSSION THREAD

**Discussion by alexmgriffiths** [jaegertracing/jaeger#5432] [Q&A] [answered]
All my spans are the same color and I was hoping I could change them depending on the tag I set.

I've tried setting a decent amount of tags like

	span.SetTag("component", "Merchants")
	span.SetTag("service", "service")
	span.SetTag("name", "service")
	span.SetTag("operation_name", "service")
	span.SetTag("operation-name", "service")
	span.SetTag("operation", "service")

With no luck, any guidance here is greatly appreciated

![image](https://github.com/jaegertracing/jaeger/assets/67096118/3f82ca8f-49b6-4d61-b5c0-4d339a1ac59c)

**Comment by yurishkuro** [comment, marked as answer]
Colors are used to distinguish different service names. I think using different colors when ALL spans in the trace belong the same service could be a reasonable enhancement, but Jaeger is optimized for _distributed_ tracing, not single service profiling.

**Reply by alexmgriffiths** [reply]
Okay, thank you
